src/main/java/in/ashwanthkumar/gocd/slack/ruleset/Rules.java

```diff
@@ -199,6 +220,11 @@ public static Rules fromConfig(Config config) {
             displayMaterialChanges = config.getBoolean("displayMaterialChanges");
         }
 
+        boolean processAllRules = false;
+        if (config.hasPath("processAllRules")) {
```

Comment:
I hate to say this, but can you please switch to `-` based convention for the config name? I wanted to do the same for `displayMaterialChanges` as well. Pretty please!